app/core/config.py

```
@@ -0,0 +1,293 @@
+        description="Application name"
+    )
+    environment: str = Field(
+        default="development",
+        description="Deployment environment"
+    )
+    debug: bool = Field(
+        default=False,
+        description="Debug mode flag"
+    )
+    host: str = Field(
+        default="0.0.0.0",
```

Ruff (0.11.9)

Possible binding to all interfaces

(S104)